parent/pom.xml

```diff
@@ -247,7 +247,7 @@
                 <repository>
                     <id>sonatype-nexus-snapshots</id>
                     <name>Sonatype Nexus Snapshots</name>
-                    <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+                    <url>https://jakarta.oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
Any reason why we shouldn't use the existing sonatypeOssDistMgmtSnapshotsUrl property?

Reply:
+1, although note trailing slash, which may or may not matter.